src/commands/manifest.ts

```diff
@@ -16,15 +15,15 @@ export default class Manifest extends Command {
     try {
       fs.unlinkSync('oclif.manifest.json')
     } catch {}
-    const {args} = this.parse(Manifest)
+    const {args} = await this.parse(Manifest)
     const root = path.resolve(args.path)
-    let plugin = new Config.Plugin({root, type: 'core', ignoreManifest: true, errorOnManifestCreate: true})
+    let plugin = new Plugin({root, type: 'core', ignoreManifest: true, errorOnManifestCreate: true})
     if (!plugin) throw new Error('plugin not found')
     await plugin.load()
     if (!plugin.valid) {
       const p = require.resolve('@oclif/plugin-legacy', {paths: [process.cwd()]})
       const {PluginLegacy} = require(p)
-      delete plugin.name
+      // delete plugin.name
```

Comment:
Does this need to be removed? uncommented?

Reply:
I'll remove. Not sure if it is needed but it didn't exist in core and didn't seem to matter.